### PR TITLE
Setup/reactcore nestcore

### PR DESCRIPTION
**Pull request by ckhandla94**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,19 @@
+APP_ENV=local
```
[the remark this answers is not in the document]
Comment (Kishanackplus) [reply .env]: done

File: apps/admin/src/app/AppRoutes.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { LoadingScreen, useAuth } from '@mlm/react-core';
+import Routes from './routes';
+
+const AppRoutes = () => {
```
[the remark this answers is not in the document]
Comment (Kishanackplus) [reply apps/admin/src/app/AppRoutes.tsx:4]: done

File: apps/admin/src/app/layout/dashboard/header/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,150 @@
+import { useTheme } from '@mui/material/styles';
```
[the remark this answers is not in the document]
Comment (Kishanackplus) [reply apps/admin/src/app/layout/dashboard/header/Header.tsx:1]: done

File: apps/admin/src/app/layout/dashboard/navbar/Navbar.tsx [hunk 4]
```diff
@@ -0,0 +1,117 @@
+'use client';
```
[the remark this answers is not in the document]
Comment (Kishanackplus) [reply apps/admin/src/app/layout/dashboard/navbar/Navbar.tsx:1]: done

File: apps/admin/src/app/sections/auth/verify-otp/Index-verify.tsx
[the remark this answers is not in the document]
Comment (ajay-khandla) [reply apps/admin/src/app/sections/auth/verify-otp/Index-verify.tsx:1]: @kishanKhandala not solved

File: apps/admin/src/app/pages/user/user-list.tsx [hunk 5]
```diff
@@ -0,0 +1,339 @@
...
+  //         //nothing.
+  //       });
+  //   },
+  //   [deleteConfirm, showToasty],
+  // )
+
+  // const handleDeleteUser = useCallback(
+  //   (row: IUser) => {
+  //     deleteConfirm(
+  //       {
```
[the remark this answers is not in the document]
Comment (ajay-khandla) [reply apps/admin/src/app/pages/user/user-list.tsx]: @kishanKhandala Not resolved

File: apps/admin/src/app/layout/auth/index.tsx [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  height: '100vh',
+  overflow: 'auto',
+});
+
+
+export default function AuthLayout() {
+
+  return (
+    <RootStyle>
+      {/* <RightTopBackground />
```
[the remark this answers is not in the document]
Comment (kishanKhandala) [reply apps/admin/src/app/layout/auth/index.tsx]: done

**Pull request merged**